### DISCUSSION TITLE
How to change screen resolution of ChromeOS?

### DISCUSSION THREAD

**Discussion by maknika39** [sebanc/brunch#1984] [Q&A]
I have installed ChromeOS on my laptop using Brunch and everything works well. My laptop screen supports 3840x2160 and that's what ChromeOS runs in as well but my question is how do I change it to something lower like 1920x1080? On the larger resolution the laptop gets very warm and the battery life isn't very long so I would like to lower the resolution. How would I go about doing this? Also, I am talking about the screen resolution, not the slider in the ChromeOS settings that changes the scale of the UI.

**Comment by RealAct** [comment]
If your device supports different resolutions, simply open Settings >> Device >> Display >> Resolution.
